exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -81,6 +83,11 @@
       new RangeLongValidator("planner.identifier_max_length", 128 /* A minimum length is needed because option names are identifiers themselves */,
                               Integer.MAX_VALUE, DEFAULT_IDENTIFIER_MAX_LENGTH);
 
+  public static final OptionValidator FILTER_MIN_SELECTIVITY_ESTIMATE_FACTOR = new MinRangeDoubleValidator("planner.filter.min_selectivity_estimate_factor",
```

Comment:
This will reduce the flexibility for the user. I think we should still maintain one-on-one mappings for RelMdRowCount, RelMdSelectivity and the selectivity upper/lower bounds. Selectivity and bounds will be different e.g. for filters than for joins.